src/Mapbender/CoreBundle/Command/DatabaseUpgradeCommand.php

```diff
@@ -13,9 +13,9 @@ class DatabaseUpgradeCommand extends ContainerAwareCommand {
 
     protected function configure() {
         $this
-            ->setHelp('The <info>mapbender:database:upgrade</info> command updates the Datesbase to the new schema of mapbender version 3.0.6')
+            ->setHelp('The <info>mapbender:database:upgrade</info> command updates the database to the new schema of mapbender version')
```

Comment:
Maybe also delete the word "version"...otherwise, looks good to me!

Reply:
changed the text to your mapbender version